doc/examples/tissue_classification.py

```diff
@@ -4,10 +4,10 @@
 Tissue Classification of a T1-weighted Structural Image
 =======================================================
 
-This example explains how to segment a T1-weighted structural image by using a
+This example explains how to segment a T1-weighted structural image by using 
 Bayesian formulation. The observation model (likelihood term) is defined as a
-Gaussian distribution and a Markov Random Field (MRF) is used to model the a
-priori probability of the context-dependent patterns of the different tissue
+Gaussian distribution and a Markov Random Field (MRF) is used to model the 
+apriori probability of context-dependent patterns of different tissue
```

Comment:
_a priori_ is spelled as two words, and although I am not a native English speaker I do not think the articles are necessary:

Reply:
Corrected it. Thank you @jhlegarreta . But other changes are valid .